todo_app/app.py

```diff
@@ -1,15 +1,24 @@
-from flask import Flask
+from flask import Flask, render_template, redirect, url_for, request
 
 from todo_app.flask_config import Config
 
+from todo_app.data.session_items import get_items, add_item
+
 app = Flask(__name__)
 app.config.from_object(Config)
 
 
-@app.route('/')
+@app.route('/index')
```

Comment:
You should update the README as currently devs would have no idea to go to /index without reading the code. 
The README should enable people to clone the repo and run it without reading the code itself.